### PR TITLE
Added Amit Khatri profile

### PR DESCRIPTION
**Pull request by aamita96**
New aamita96.json file added to this repo. 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/355"></a>

### CODE REVIEW

**Review by voidz7 — CHANGES_REQUESTED** [review]

File: public/data/aamita96.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "url": "https://www.linkedin.com/in/amit-khatri-193ab9125/",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Follow me on Instagram",
+        "url": "https://www.instagram.com/amit_k96/",
+        "icon": "instagram"
+      }
+    ]
+  }
```
[the remark this answers is not in the document]
Comment (aamita96) [reply public/data/aamita96.json]: You mean aligning the parenthesis right? If that is the case I have done it, If not then please specify. Thanks.

[the remark this answers is not in the document]
Comment (voidz7) [reply public/data/aamita96.json]: yep .

**Review by voidz7 — APPROVED** [review]

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
GitHub is highlighting the file ending is not correct

**Comment by aamita96** [comment]
> GitHub is highlighting the file ending is not correct

@eddiejaoude would be great if you can specify what I am missing like it was whitespace or new-line. I have tried to keep the file like your provided JSON and tried to compare other JSON files as well didn't find any major diferrence.

**Review by Kunal0007 — APPROVED** [review]
LGTM

**Review by MukulKolpe — APPROVED** [review]

**Review by rohan-kulkarni-25 — APPROVED** [review]
LGTM 💥

**Review by eddiejaoude — APPROVED** [review]
Looks good now 👍

**Pull request merged**